core/shared/src/main/scala/bobcats/Hotp.scala

```diff
@@ -38,11 +43,19 @@ object Hotp extends HotpCompanionPlatform {
 
   implicit def forSync[F[_]](implicit F: Sync[F], H: Hmac[F]): Hotp[F] =
     new UnsealedHotp[F] {
+      override def generate(
+          key: SecretKey[HmacAlgorithm.SHA1.type],
+          movingFactor: Long): F[Int] =
+        generate(key, movingFactor, digits = 6)
```

Comment:
I think we can put this default implementation in the parent trait actually :)

Reply:
Yes you're absolutely right, d'oh!